sig-auth/README.md

```diff
@@ -37,6 +37,84 @@ The Chairs of the SIG run operations and processes governing the SIG.
 * [Mailing list](https://groups.google.com/forum/#!forum/kubernetes-sig-auth)
 * [Open Community Issues/PRs](https://github.com/kubernetes/community/labels/sig%2Fauth)
 
+## Subprojects
```

Comment:
Not sure what section `k8s.io/kubernetes/pkg/kubectl/cmd/auth` would go under.  Maybe have a dedicated section for `kubectl` related things and move `k8s.io/client-go/plugin/pkg/client/auth` there as well?

Reply:
A subset of `k8s.io/apiserver/pkg/endpoints/filters` is also relevant to us (impersonation, authn, authz).

Reply:
Added kubectl bits to the authorization project. I think the filters need to be broken out of the kitchen sink filter package before we can claim them.